SabreTools.FileTypes/CHD/CHDFile.cs

```diff
@@ -19,33 +22,62 @@ public abstract class CHDFile : BaseFile
         protected uint length;              // length of header (including tag and length fields)
         protected uint version;             // drive format version
 
+        protected CHDFile generated;
```

Comment:
You don't have any comments on here so I don't understand what either of these do.

Reply:
yeah, that code be done better - i had some problems with giving arguments and keeping them inside...